stacks-signer/src/signerdb.rs

```diff
@@ -826,6 +826,16 @@ impl SignerDb {
         try_deserialize(result)
     }
 
+    /// Return the count of globally signed blocks in a tenure (identified by its consensus hash)
+    pub fn get_globally_signed_block_count_in_tenure(
```

Comment:
This function name is misleading. You are querying the number of blocks in a tenure that are _globally accepted_, not signed.  Your tests don't even check the signature.  Please instead call this function `get_globally_accepted_block_count_in_tenure()`.